### PR TITLE
B20: Handling data

### PR DESCRIPTION
**Pull request by essihe**
added video link to introduction on handling research data

### CODE REVIEW

**Comment by essihe** [comment]
This video link is from University of Sydney and it provides a basic introduction to understand why research data management is needed and should be done. I saw software carpentry also has videos on data management but I believe those are better for section C14 as the videos seem to be more detailed.

File: courses/B.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@
     becomes unusable.  There are good and bad ways to do this.
   reading: |
     `Aalto scicomp data docs <https://scicomp.aalto.fi/data/>`__
```
Comment (rkdarst) [inline courses/B.yaml]: I guess it's worth just going ahead and removing this... it'll probably never be good enough.

File: courses/B.yaml [hunk 2]
```diff
@@ -4,6 +4,8 @@
...
+  video: |
+    `Research Data Management 101 <https://www.youtube.com/watch?v=_pHVMwfOZf8>`_
```
Comment (rkdarst) [inline courses/B.yaml]:
```suggestion
    `What is not data management, in 5 minutes <https://www.youtube.com/watch?v=N2zK3sAtr-4>`_
    `Research Data Management 101 <https://www.youtube.com/watch?v=_pHVMwfOZf8>`_
```
This is one of my favorite ways to show what is data management - by showing the opposite of it.  Very relateable that way.

File: sites/aalto/aalto.yaml [hunk 3]
```diff
@@ -6,3 +6,6 @@ A01-local: |
 B14-local: |
   `Aalto scicomp <https://scicomp.aalto.fi/data/>`__
   `Aalto RDM <https://aalto.fi/rdm>`__
```
Comment (rkdarst) [inline sites/aalto/aalto.yaml]: I notice that this is nothing - somehow it's a semi-duplicate of C14, which we talked about.  Can be removed.
```suggestion
```

**Review by rkdarst — APPROVED** [review]
Looks good to me.  I have a feeling we could find a better overview video, but... for all I've seen, I don't know anything better!  I asked in another channel, who knows if we get something...

**Comment by rkdarst** [comment]
I'll leave you to accept the suggestions, or not, and merge.  thanks!

**Pull request merged**